R/equation.R

```diff
@@ -0,0 +1,16 @@
+#' Model equations
+#'
+#' Display the mathematical representation of a fitted model.
+#'
+#' @param object A fitted model object.
+#' @param ... Other arguments passed to methods
+#'
+#' @return LaTeX output suitable for usage within R Markdown documents.
```

Comment:
Also make this more general by saying something like

> Markup output suitable for rendering the equation.

Reply:
Also fine. Ideally `equation` methods would support more than just LaTeX (such as the print flexibility in [SymPy](https://docs.sympy.org/latest/tutorial/printing.html)), however it would be nice to impose a consistent default.